packages/edge/package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "@llamaindex/edge",
+  "version": "0.1.21",
+  "license": "MIT",
+  "type": "module",
+  "dependencies": {
+    "@anthropic-ai/sdk": "^0.15.0",
```

Comment:
best we generate this file from core/package.json and copy its dependencies and version